Doc/library/csv.rst

```diff
@@ -95,11 +95,11 @@ The :mod:`csv` module defines the following functions:
    :func:`list_dialects` function.  The other optional *fmtparams* keyword arguments
    can be given to override individual formatting parameters in the current
    dialect.  For full details about dialects and formatting parameters, see
-   the :ref:`csv-fmt-params` section. To make it
+   the :ref:`csv-fmt-params` section.  To make it
    as easy as possible to interface with modules which implement the DB API, the
    value :const:`None` is written as the empty string.  While this isn't a
    reversible transformation, it makes it easier to dump SQL NULL data values to
-   CSV files without preprocessing the data returned from a ``cursor.fetch*`` call.
+   CSV files without preprocessing the data returned by a ``cursor.fetch*`` call.
```

Comment:
`from` is not incorrect here

Reply:
Yet ‘returned by’ is used six times in the page, and ‘returned from’ only twice so are you sure it is not correct? In this case the six occurrences should be fixed.

Reply:
they are both correct, there is no problem to be fixed!

Reply:
Alright, I am removing this change.